README.md

```diff
@@ -23,6 +24,15 @@ In this process, there are 3 parties involved:
 * **The backend** - the shop server.
 * **Paypal-integration** - hosted service (this repository) which exposes public endpoints 
 
+
+## Architectural Diagram
+![paypal-plus-architecture](docs/paypal-plus-architecture.jpg)
```

Comment:
Below we have this section: https://github.com/commercetools/commercetools-paypal-plus-integration/blob/3cc525385d602a6414a838b6122c3e168a3e147d/README.md#how-to-use

Can you update the arrows to match the points there? Also there is a point `Backend extracts PayPal specific parameters: paymentId, PayerID and POSTs them in the request body to Paypal-integration for payment execution. Example:`. Is this reflected in the diagram?

Reply:
I refer to the mentioned document and paypal-plus official integration guide to illustrate the arrows. Please let me know which part you found incorrect. And would update new diagram to reflect `Backend extracts PayPal specific parameters: paymentId, PayerID and POSTs them in the request body to Paypal-integration for payment execution`

Reply:
It looks good now. One thing I'm missing is the last point saying `payment authorization page`. It seems to be the diagram/description is not finished. User should be redirected to the order confirmation page to confirm that order was successfully created. Can you check this and add this point so it is complete?

Reply:
Updated diagram https://github.com/commercetools/commercetools-paypal-plus-integration/pull/207/commits/3d114748b1e0d4d263ca6122b62f4bf78cd1bb14
Updated steps 
https://github.com/commercetools/commercetools-paypal-plus-integration/pull/207/commits/859bddb65c9cdeec62358d83e3671a75b900cedc
https://github.com/commercetools/commercetools-paypal-plus-integration/pull/207/commits/8d9816b2dc5f0eeb4967ed404b295ce1936f6221

Reply:
It's a bit weird because the numbers 4 and 5 are in the flipped order if you read the diagram from top to bottom. But I also don't know how to improve it. Maybe you can ask the team later in the sprint review to see if the diagram is good?